ocs_ci/framework/pytest_customization/marks.py

```diff
@@ -67,6 +67,7 @@
 flowtests = pytest.mark.flowtests
 performance = pytest.mark.performance
 scale = pytest.mark.scale
+regression = pytest.mark.regression
```

Comment:
please rename this to just "scale" for clarity

Reply:
> > > > what is the purpose for this one?
> > > 
> > > 
> > > This marker will be used to run scale regression tests. We should name as "scale_regression" to make it more clear about the purpose of the marker.
> > 
> > 
> > Just "scale" should be sufficient in that case
> 
> We already have scale label for all scale tests. We need a subset of scale test to run as scale regression.

I see that, I think we could use pytest filter to achieve what we need, I really dont see need for another marker here if its just one test?

Reply:
its not just one test, i added small group for test for fast jenkins run, adding the rest of  tests